components/ResultsPageComponents/TopThree/TopThree.jsx

```diff
@@ -15,24 +15,32 @@ export default function TopThree({
   }
 
   return (
-    <Flex direction="column">
-      <Flex direction="column" align="flex-start">
-        <Text
-          fontWeight={600}
-          fontSize="33px"
-          lineHeight="37px"
-          py="15px"
-          align="center"
-        >
+    <Flex
+      direction="column"
+      // minWidth="300px"
```

Comment:
Hey @Amutha37 , well done, I like how you have both mobile and desktop options. Should we remove those `//` attributes completely if you no longer need them?

Reply:
yups, agree with @OlaShabalina ,

commented could cause confusion, and encourage more commented out code from future contributors!
Let's clean them up, if they are not needed anymore. 
I can see other commented out code in other files in this PR; please remove them too 🙇🏽